<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationProcessDialog</class>
 <widget class="QDialog" name="RegistrationProcessDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="processDescription">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Registration Process Running</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="elapsedTime">
     <property name="text">
      <string>Running Time: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusMessage">
     <property name="text">
      <string>Left Status</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="statusBoxTextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::FixedColumnWidth</enum>
     </property>
     <property name="lineWrapColumnOrWidth">
      <number>250</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
